public/docs/ts/latest/guide/router.jade

```diff
@@ -1,8 +1,1515 @@
+
+  We cover the router's primary features in this chapter, illustrating them through the evolution
+  of a small application that we can [run live](/resources/live-examples/router/ts/plnkr.html).
+.l-sub-section
+  img(src='/resources/images/devguide/plunker-separate-window-button.png' alt="pop out the window" align="right" style="margin-right:-20px")
+  :marked
+    To see the URL changes in the browser address bar,
+    pop out the preview window by clicking the blue 'X' button in the upper right corner.
+
+.l-main-section
+:marked
+  ## Overview
+
+  The browser is a familiar model of application navigation.
+  We enter a URL in the address bar and the browser navigates to a corresponding page.
+  We click links on the page and the browser navigates to a new page.
+  We click the browser's back and forward buttons and the browser navigates
+  backward and forward through the history of pages we've seen.
+
+  The Angular ***Component Router*** ("the router") borrows from this model.
+  It can interpret a browser URL as an instruction
+  to navigate to a client-generated view and pass optional parameters along to the supporting view component
+  to help it decide what specific content to present.
+  We can bind the router to links on a page and it will navigate to
+  the appropriate application view when the user clicks a link.
+  We can navigate imperatively when the user clicks a button, selects from a drop box,
+  or in response to some other stimulus from any source. And the router logs activity
+  in the browser's history journal so the back and forward buttons work as well.
+
+  We'll learn many router details in this chapter which covers
+
+  * Setting the [base href](#base-href)
+  * Importing from the [router library](#import)
+  * [configuring the router](#route-config)
+  * the [link parameters array](#link-parameters-array) that propels router navigation
+  * navigating when the user clicks a data-bound [RouterLink](#router-link)
+  * navigating under [program control](#navigate)
+  * embedding critical information in the URL with [route parameters](#route-parameters)
+  * add [child routes](#child-router) under a feature section
+  * setting an [index route](#index) as the default
+  * confirming or canceling navigation with [guards](#guards)
+  * passing optional information in [query parameters](#query-parameters)
+  * choosing the "HTML5" or "hash" [URL style](#browser-url-styles)
+
+  We proceed in phases marked by milestones building from a simple two-pager with placeholder views
+  up to a modular, multi-view design with child routes.
+
+  But first, an overview of router basics.
+
+.l-main-section
+:marked
+  ## The Basics
+  Let's begin with a few core concepts of the Component Router.
+  Then we can explore the details through a sequence of examples.
+
+:marked
+  ### *&lt;base href>*
+  Most routing applications should add a `<base>` element to the **`index.html`** just after the  `<head>` tag
```

Comment:
This might be confusing. It can be interpreted it as 

```
<head>
   stuff
</head>
<base ...>
```
but it should rather be 
```
<head>
  <base ...>
   stuff
</head>
```
I would write "as first child in `<head>`"